config/locales/forms/en.yml

```
@@ -125,9 +125,8 @@ en:
       continue: Continue
       instructions_text: Press the button on your security key to register it with %{app_name}
       instructions_title: Connect your security key
-      intro_html: Add a security key as an authentication method to your
-        account.  Your security key must support the FIDO standard.  You can add
-        as many security keys as you want, and we recommend at least two for
-        easier account recovery.
+      intro_html: Add a security key that meets the FIDO standard as your
+        authentication method. You can add as many security keys as you want. To
+        get started, first give your security key a nickname.
```

this copy no longer seems to have any HTML in it, should we rename the key to just be `intro`?
```suggestion
      intro: Add a security key that meets the FIDO standard as your
        authentication method. You can add as many security keys as you want. To
        get started, first give your security key a nickname.
```